packages/front-end/components/Settings/AthenaForm.tsx

```diff
@@ -24,9 +24,13 @@ const AthenaForm: FC<{
                 value: "auto",
                 display: "Auto-discovery",
               },
+              {
+                value: "assumeRole",
+                display: "Assume IAM Role",
+              },
             ]}
-            helpText="'Auto-discovery' will look for credentials in environment variables and instance metadata."
-            value={params.authType || "accessKey"}
+            helpText="'Auto-discovery' will look for credentials in environment variables and instance metadata. 'Assume IAM Role' uses the current role to assume another role and execute Athena with temporary credentials."
+            value={params.authType || "auto"}
```

Comment:
Change this default back to `accessKey` since that was the default before we introduced the `authType` param.

Reply:
Will do

Reply:
done